### PR TITLE
Feat/improve autodiag

### PR DESCRIPTION
**Pull request by euhmeuh**
See MBE-114, MBE-102, MBE-125 and MBE-128.

### CODE REVIEW

File: src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml [hunk 1]
```diff
@@ -182,6 +183,11 @@
 
    <translation id="tools_lineDiagnostics_solution_proposal_step_validationRefused">La demande d'intervention a été refusée par l'assistance technique OVH.</translation>
 
+   <!-- Modal -->
+   <translation id="tools_lineDiagnostics_confirm_modal_title">Confirmation de demande d'intervention</translation>
+   <translation id="tools_lineDiagnostics_confirm_modal_explanation">Afin de finaliser votre demande d'intervention, nous allons réalisé un test technique qui rendra votre <strong>connexion internet indisponible pendant 15 minutes</strong> environ. Merci de prendre toutes les précautions nécessaires à ce cas avant de continuer.</translation>
```
Comment (FredericEspiau) [inline src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml]: Please avoid having `html` in `xml` files

File: src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -179,6 +183,11 @@
...
+   <translation id="tools_lineDiagnostics_confirm_modal_unplug_modem_text">Immédiatement après avoir cliquer sur Continuer, vous devez débrancher votre modem de votre prise téléphonique.</translation>
```
Comment (FredericEspiau) [inline src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml]:
```suggestion
   <translation id="tools_lineDiagnostics_confirm_modal_unplug_modem_text">Immédiatement après avoir cliqué sur {{continueButtonLabel}}, vous devez débrancher votre modem de votre prise téléphonique.</translation>
```

File: src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html [hunk 3]
```diff
@@ -21,24 +21,38 @@
...
+                <span data-translate="tools_lineDiagnostics_detection_step_modem_question"></span>
+                <oui-popover>
+                    <button type="button"
+                            class="oui-popover-button"
+                            oui-popover-trigger></button>
+                    <oui-popover-content>
+                        <p class="text-center"
+                            data-translate="tools_lineDiagnostics_detection_step_modem_question_help">
+                        </p>
+                        <img src="assets/images/diagnostic/broadband-light.png" data-ng-attr-alt="{{:: $ctrl.currentAction }}" />
```
Comment (FredericEspiau) [inline src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html]:
```suggestion
                        <img src="assets/images/diagnostic/broadband-light.png" data-ng-attr-alt="{{:: $ctrl.currentAction }}" aria-hidden="true" />
```

No sure though

File: src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html [hunk 4]
```diff
@@ -21,24 +21,38 @@
...
+                    </oui-popover-content>
+                </oui-popover>
+            </label>
+            <oui-radio-toggle-group class="d-inline-block"
+                                    data-model="$ctrl.currentLineDiagnostic.data.answers.modemIsSynchronized"
+                                    data-on-change="$ctrl.answerQuestion('modemIsSynchronized', modelValue)"
+                                    data-ng-hide="$ctrl.loading">
```
Comment (FredericEspiau) [inline src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html]:
```suggestion
                                    data-ng-if="$ctrl.loading">
```
?

File: src/ovh-angular-line-diagnostics/ovh-angular-line-diagnostics.controller.js [hunk 5]
```diff
@@ -222,6 +223,15 @@ angular.module("ovh-angular-line-diagnostics").controller("LineDiagnosticsCtrl",
         this.startPoller();
     }
 
+    confirmIntervention () {
+        this.openDialog("ovh-angular-line-diagnostics-confirm-intervention-dialog")
+            .then((result) => {
```
Comment (FredericEspiau) [inline src/ovh-angular-line-diagnostics/ovh-angular-line-diagnostics.controller.js]: What kind of result is that ?

**Review by FredericEspiau — CHANGES_REQUESTED** [review]

File: src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml (hunk 1 above)
Comment (euhmeuh) [reply src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml]: I know right, but I'm definitely not gonna split that sentence into meaningless chunks :/

Comment (FredericEspiau) [reply src/ovh-angular-line-diagnostics/translations/Messages_fr_FR.xml]: True, this is an issue. I'd love to be able to use markdown in the translations

File: src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-solution-proposal-step.html [hunk 6]
```diff
@@ -349,7 +349,7 @@
 
                 <oui-button data-disabled="!requestInterventionForm.$valid"
                             data-on-click="$ctrl.sendInterventionForm()"
-                            data-text="{{:: 'tools_lineDiagnostics_solution_proposal_step_intervention_request_continue' | translate }}"
+                            data-text="{{:: 'tools_lineDiagnostics_continue' | translate }}"
```
Comment (marie-j) [inline src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-solution-proposal-step.html]: `text` is deprecated, prefer using a nested `span` if possible

File: src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html [hunk 7]
```diff
@@ -21,24 +21,38 @@
...
+                    <span data-translate="tools_lineDiagnostics_detection_step_modem_light_on"></span>
+                </oui-radio>
+                <oui-radio data-value="false">
+                    <span data-translate="tools_lineDiagnostics_detection_step_modem_light_blink"></span>
+                </oui-radio>
+                <oui-radio data-value="false">
+                    <span data-translate="tools_lineDiagnostics_detection_step_modem_light_off"></span>
+                </oui-radio>
+            </oui-radio-toggle-group>
+            <p data-ng-if="$ctrl.loading">
```
Comment (FredericEspiau) [inline src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html]: More like a `div` if it doesn't contain only text

File: src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html [hunk 8]
```diff
@@ -21,24 +21,38 @@
...
-                    <oui-radio data-value="false">
-                        <span data-translate="tools_lineDiagnostics_detail_no"></span>
-                    </oui-radio>
-                </oui-radio-toggle-group>
-                <p data-ng-if="$ctrl.loading">
-                    <oui-spinner data-size="s"></oui-spinner>
-                    <span class="ml-2" data-translate="tools_lineDiagnostics_please_wait"></span>
-                </p>
-            </oui-field>
+            <label class="oui-label">
```
Comment (FredericEspiau) [inline src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html]: There are more than one `labelable` element in this `label`, please use `for="id"` instead

**Review by FredericEspiau — APPROVED** [review]

File: src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html (hunk 8 above)
Comment (FredericEspiau) [reply src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html]: `id="radioLabel"`

File: src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html [hunk 9]
```diff
@@ -21,24 +21,38 @@
...
+                            oui-popover-trigger></button>
+                    <oui-popover-content>
+                        <p class="text-center"
+                            data-translate="tools_lineDiagnostics_detection_step_modem_question_help">
+                        </p>
+                        <img src="assets/images/diagnostic/broadband-light.png" data-ng-attr-alt="{{:: $ctrl.currentAction }}" aria-hidden="true" />
+                    </oui-popover-content>
+                </oui-popover>
+            </label>
+            <oui-radio-toggle-group class="d-inline-block"
```
Comment (FredericEspiau) [inline src/ovh-angular-line-diagnostics/steps/ovh-angular-line-diagnostics-detection-step.html:38]: `aria-labelledby="radioLabel"`

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**